apis/spanner/v1beta1/instance_reference.go

```diff
@@ -168,6 +168,10 @@ func parseSpannerInstanceExternal(external string) (parent *SpannerInstanceParen
 	return parent, resourceID, nil
 }
 
+func (r *SpannerInstanceRef) SpannerInstanceConfigPrefix() string {
+	return fmt.Sprintf("projects/%s/instanceConfigs/", r.parent.ProjectID)
```

Comment:
hmmm, is this the same as `r.parent`?

Reply:
not actually, the parent of instance is just projects/<project>. The instance config is a different pseudo resource. The Create function need the config field in this form to work.